src/coreclr/jit/llvm_types.h

```diff
@@ -47,18 +70,24 @@ struct FieldDesc
     {
         return m_corType;
     }
+
+    bool isGcPointer()
+    {
+        return m_isGcPointer;
+    }
```

Comment:
```suggestion
    bool isGcPointer()
    {
        return m_corType == CORINFO_TYPE_CLASS || m_corType == CORINFO_TYPE_BYREF;
    }
```
Seems like there's no need to cache it.